Dockerfile

```diff
@@ -0,0 +1,28 @@
+# This dockerfile is only meant for local development of Misago
+# If you are looking for a proper docker setup for Misago look elsewhere
+FROM python:3
+
+# Install dependencies in one single command/layer
+RUN apt-get update && apt-get install -y \
+    vim \
+    libffi-dev \
+    libssl-dev \
+    sqlite3 \
+    libjpeg-dev \
+    libopenjpeg-dev \
+    locales \
+    cron \
+    postgresql-client
+
+# Add requirements and install them. We do this unnecessasy rebuilding.
+ADD requirements.txt /
+RUN pip install -r requirements.txt
+
+# Add project source and run
+# ADD . /srv/misago (We don't need this in development, leaving out for now)
```

Comment:
If its not needed, then lets not leave it commented out to make people wonderif maybe its needed now ;)